### ISSUE TITLE
REQUEST: allow dropping tag to other player to tag

### ISSUE THREAD

**Issue by jeraminl** [SuperDyl/tag-game#2]
I have party plugins that disables friendly fire, would be nice if you can work around the need to physically hit player with the tag.

**Comment by SuperDyl** [comment]
That's a really good idea. By default, I want the pack to mostly match the features of Vanilla Tweaks, so I don't think I can make it the default. Perhaps I could add an optional setting to allow tagging players by dropping the tag and the other player picking it up? It would be similar to how the Hermits originally played tag

**Comment by jeraminl** [comment]
yep! maybe there's an even better way to bypass the disabled pvp like right-clicking? i'm not sure what is possible

**Comment by SuperDyl** [comment]
That could be possible, but there would be limitations. If I used custom enchantments, it would mark the world experimental which would make it difficult to remove the datapack later. If I retextured a carrot on a stick, it would require a resource pack alongside the datapack. If I made the tag item count as a food, I could test for the clicking, but then I have to calculate if the second player can be clicked on. Finally, I could make holding the tag item create an interaction entity, but that would make it impossible to interact with things like chests while holding the tag.

Basically, it’s possible with trade offs. I would only want to make one trade off here and I’m leaning towards dropping the item on the ground and the other player picking it up, mostly because I like the similarity to the original tag game.

What do you think?

**Comment by jeraminl** [comment]
I think the interaction entity thing sounds perfect. I don't think not being able to interact while holding it is a problem as you'd probably be focused on tagging people.
